addons/units/CfgVehicles_PublicUnits.hpp

```diff
@@ -194,27 +204,26 @@ class CLASS(Unit_I_Specialist): CLASS(Unit_Combat_RS_BS_GP_BB) {
     respawnWeapons[] = {"LMG_Mk200_MRCO_F", "hgun_Rook40_F", "Binocular", "Throw", "Put"};
     magazines[] = {"200Rnd_65x39_cased_Box_Tracer", ITEMS_2(16Rnd_9x21_Mag), ITEMS_2(SmokeShell), ITEMS_2(HandGrenade)};
     respawnMagazines[] = {"200Rnd_65x39_cased_Box_Tracer", ITEMS_2(16Rnd_9x21_Mag), ITEMS_2(SmokeShell), ITEMS_2(HandGrenade)};
-    linkedItems[] = {DEFAULT_ITEMS, QCLASS(Cap_BlackLogo), "G_Shades_Black", QCLASS(Vest_PlateCarrierFull_Black)};
-    respawnLinkedItems[] = {DEFAULT_ITEMS, QCLASS(Cap_BlackLogo), "G_Shades_Black", QCLASS(Vest_PlateCarrierFull_Black)};
+    linkedItems[] = {DEFAULT_ITEMS, "H_HelmetB", "G_Shades_Black", QCLASS(Vest_PlateCarrier_Green)};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "H_HelmetB", "G_Shades_Black", QCLASS(Vest_PlateCarrier_Green)};
     backpack = QCLASS(Backpack_AssaultExpanded_Green_Specialist_Filled);
 
     icon = "iconManMG";
 
     headgearList[] = {
-        QCLASS(Cap_BlackLogo), 1,
-        QCLASS(Cap_TanLogo), 1,
-        QCLASS(Cap_Earpiece_BlackLogo), 1,
-        QCLASS(Cap_Earpiece_TanLogo), 1,
-        QCLASS(Cap_Backwards_BlackLogo), 1,
-        QCLASS(Cap_Backwards_TanLogo), 1
+        "H_HelmetB", 1,
```

Comment:
Put vanilla ones at the end for organization. Try to sort mod ones same as they are defined in the mod.

Reply:
Done